Library/Formula/ansible-cmdb.rb

```diff
@@ -0,0 +1,24 @@
+class AnsibleCmdb < Formula
+  desc "Ansible-cmdb takes the output of Ansible's fact gathering and converts it into a static HTML overview page containing system configuration information."
+  homepage "https://github.com/fboender/ansible-cmdb"
+  url "https://github.com/fboender/ansible-cmdb/releases/download/1.6/ansible-cmdb-1.6.zip"
+  version "1.6"
```

Comment:
Suspect this is redundant. We can handle reading simple URLs for version numbers.